docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -45,6 +45,13 @@ INSERT INTO [ TABLE ] table_identifier [ partition_spec ]
 
     **Syntax:** `PARTITION ( partition_col_name  = partition_col_val [ , ... ] )`
 
+* **column_list**
+
+  An optional parameter that specifies a comma separated list of columns belong to the `table_identifier`.
```

Comment:
nit: 
 - `comma separated` -> `comma-separated`
 -  `belong` -> `belonging`
 - `the table_identifier` -> `the table_identifier table`